### PR TITLE
Update the US Federal Gov site notice on cg-site

### PR DESCRIPTION
**Pull request by rememberlenny**
open-close

### CODE REVIEW

**Review by el-mapache — DISMISSED** [review]
One nitpick but looks fine otherwise.

File: static/assets/js/index.js [hunk 1]
```diff
@@ -9,12 +12,37 @@
...
+      }
+    });
+    $govBarWrapper.on('click', function(){
+      toggleGovBar();
+    })
     ajaxifyContactForm();
   });
 
+  function toggleGovBar() {
+    if ($('#gov-banner').hasClass('hide')) {
```
Comment (el-mapache) [inline static/assets/js/index.js]: You can use the `$govBarContent` variable here I think.

Comment (rememberlenny) [reply static/assets/js/index.js]: Updated.

**Review by el-mapache — APPROVED** [review]
all the thumbs up

**Pull request merged**